btk/draw_blends.py

```diff
@@ -61,6 +62,33 @@ def get_size(pixel_scale, catalog, i_obs_cond):
     return Column(size, name="size")
 
 
+def make_wcs(pix, shape, center_pix=None, center_sky=None, naxis=2):
+    """Creates wcs for an image
+    
+    Args:
+        pix (float): pixel size in arcseconds
+        shape (tuple): shape of the image
+        center_pix (tuple): position of the reference pixel used as the center of the
+                            affine transform for the wcs.
+        center_sky (float):
+        naxis (int):
+
+    Returns:
+        wcs: WCS
+    """
```

Comment:
where in the code are we making the choice for arcsecs?

Reply:
Actually there is no particular choice given to the WCS ; simply we give a pixel scale of 0.2 (for LSST) which we interpret as 0.2 arcsec, and which the wcs interprets as 0.2 degrees ; now that I think of it, it should have consequences on the result we get.  I suggest we switch to degrees by dividing the pixel scale by 3600, meaning that center_sky has to be provided in degrees and use of the WCS will require inputs or outputs in degrees, as per the documentation ; I believe there is no way to make it use arcseconds itself.

Reply:
After checking, the result is way more accurate. We can also convert automatically the center_sky in degrees but the WCS will use degrees anyway.

Reply:
I see think I understand, I think it's fine if we change to use degrees throughout. I remember now that the default catalog used is originally in degrees and at various points in the code we convert to arcsecs. We can also divide all pixel scales by 3600 in `config.py`. Maybe we can leave this for the next PR where we propagate the use of wcs consistently? Because it might break some parts of the code.

Reply:
> After checking, the result is way more accurate. We can also convert automatically the center_sky in degrees but the WCS will use degrees anyway.

What 'result' are you referring to?

Reply:
> What 'result' are you referring to?

dx and dy

Reply:
Ok, if it's much more accurate with degrees we should definitely do that then